src/Blocks/components/load-more/assets/index.js

```diff
@@ -0,0 +1,53 @@
+
+async function load() { 
```

Comment:
you have overcomplicated this please read the provided docs in the comment and see how Js is done on other blocks and projects